torch/csrc/dynamo/guards.cpp

```diff
@@ -265,14 +270,15 @@ static int TensorGuards_init(
       return -1;
     }
     auto tensor = THPVariable_Unpack(item);
-    std::vector<std::optional<int64_t>> tensor_dims_size =
+    std::vector<std::optional<c10::SymInt>> tensor_dims_size =
         per_tensor_dynamic_dims_sizes.empty()
-        ? wrapIntegersInOptional(tensor.sizes())
+        ? wrapIntegersInOptional(tensor.sym_sizes())
         : per_tensor_dynamic_dims_sizes[i];
-    std::vector<std::optional<int64_t>> tensor_dims_stride =
-        per_tensor_dynamic_dims_strides.empty()
-        ? wrapIntegersInOptional(tensor.strides())
+    std::vector<std::optional<c10::SymInt>> tensor_dims_stride = {};
+    tensor_dims_stride = per_tensor_dynamic_dims_strides.empty()
```

Comment:
Why not assign it directly

Reply:
Fixed, I think this was from back when I special cased nested tensor still, but I forgot to remove.